test/test_dna.py

```diff
@@ -778,3 +779,57 @@ def testMatchWithAmbiguityAndNotStrict(self):
     Ambiguous: 0''',
             matchToString(match, read1, read2, matchAmbiguous=True)
         )
+
+
+class TestFindKozakConsensus(TestCase):
+    """
+    Test the findKozakConsensus function.
+    """
+    def testOneKozakConsensus(self):
+        """
+        In a given sequence with an exact Kozak consensus sequence, the offset
+        and quality percentage should be as expected.
+        """
+        read = DNARead('id', 'ATTGCCGCCATGGGGG')
+        expectedRead = DNARead('id', 'ATTGCCGCCATGGGGG')
+        expectedKozakRead = DNAKozakRead(expectedRead, 3, 13, 100.0)
+
+        for kozakConsensus in findKozakConsensus(read):
```

Comment:
How many do you expect there to be? It's usually better, if you expect one or two or whatever, to say (in tests): `[expectedKozakRead] = list(findKozakConsensus(read))`  Can you tell me why? :-)

Reply:
Then you can test if the length of the list is as expected (whereas with a generator, as with findKozakConsensus, one read is yielded at a time, so the number of expected reads can't be computed before)